bsp/gd32vf103v-eval/drivers/drv_gpio.c

```diff
@@ -408,7 +413,7 @@ static rt_err_t gd32vf_pin_irq_enable(struct rt_device *device, rt_base_t pin,
         pin_irq_enable_mask &= ~irqmap->pinbit;
 
         eclic_irq_disable(irqmap->irqno);
-        exti_interrupt_flag_clear(EXTI_(index->pin));
+        exti_interrupt_flag_clear(EXTI_(irqindex));
```

Comment:
irqindex =-1 ?

Reply:
irqindex == -1 means that index == NULL and the function will be returned, so it will not be used.

Reply:
sorry, my mistake. And thanks to remind me :)

Reply:
format some place

Reply:
maybe you could merge to fix them?

Reply:
thank you